cockatrice/CMakeLists.txt

```diff
@@ -153,30 +153,43 @@ if(APPLE)
     set(cockatrice_SOURCES ${cockatrice_SOURCES} ${CMAKE_CURRENT_SOURCE_DIR}/resources/appicon.icns)
 ENDIF(APPLE)
 
-# Qt5
-find_package(Qt5 COMPONENTS Concurrent Multimedia Network PrintSupport Svg WebSockets Widgets REQUIRED)
-set(COCKATRICE_QT_MODULES Qt5::Concurrent Qt5::Multimedia Qt5::Network Qt5::PrintSupport Qt5::Svg Qt5::Widgets Qt5::WebSockets)
+LIST(APPEND COCKATRICE_QT_MODULES ${COCKATRICE_QT_VERSION_NAME}::Concurrent ${COCKATRICE_QT_VERSION_NAME}::Multimedia ${COCKATRICE_QT_VERSION_NAME}::Network ${COCKATRICE_QT_VERSION_NAME}::PrintSupport ${COCKATRICE_QT_VERSION_NAME}::Svg ${COCKATRICE_QT_VERSION_NAME}::Widgets ${COCKATRICE_QT_VERSION_NAME}::WebSockets)
+IF (Qt6_FOUND)
+    LIST(APPEND COCKATRICE_QT_MODULES ${COCKATRICE_QT_VERSION_NAME}::Core5Compat ${COCKATRICE_QT_VERSION_NAME}::Gui)
+ENDIF()
 
-# Qt5LinguistTools
-find_package(Qt5LinguistTools)
-if(Qt5LinguistTools_FOUND)
-    list(APPEND COCKATRICE_LIBS Qt5::LinguistTools)
+# Optional Qt LinguistTools
+IF(${COCKATRICE_QT_VERSION_NAME}LinguistTools_FOUND)
```

Comment:
it doesn't seem like you ever run find_package(qtxLinuguistTools before this is checked, neither here or in the findqt file, this will probably not work at all

Reply:
it seems like it would silently fail too, so we have no idea if it does

Reply:
I just checked in the application, the language selector is nonfunctional because of this

Reply:
Resolved this